Source/WebCore/platform/audio/cocoa/SpatialAudioExperienceHelper.mm

```diff
@@ -32,7 +32,16 @@
 
 namespace WebCore {
 
-RetainPtr<CASpatialAudioExperience> createExperienceWithOptions(const SpatialAudioExperienceOptions& options)
+static CASoundStageSize toCASoundStageSize(MediaPlayerSoundStageSize size)
+{
+    static_assert(static_cast<CASoundStageSize>(MediaPlayerSoundStageSize::Auto) == CASoundStageSizeAutomatic);
+    static_assert(static_cast<CASoundStageSize>(MediaPlayerSoundStageSize::Small) == CASoundStageSizeSmall);
+    static_assert(static_cast<CASoundStageSize>(MediaPlayerSoundStageSize::Medium) == CASoundStageSizeMedium);
+    static_assert(static_cast<CASoundStageSize>(MediaPlayerSoundStageSize::Large) == CASoundStageSizeLarge);
+    return static_cast<CASoundStageSize>(size);
```

Comment:
I think it would be a little safer to write this as a `switch` statement that `RELEASE_ASSERT_NOT_REACHED()`s in the `default` case. Right now this implementation asserts that the known enum values numerically match, but will still allow you to cast an unknown value.

Reply:
I'll address this in a follow up.